cicd/forgeops-tests/config/ProductConfig.py

```diff
@@ -138,17 +150,6 @@ def start_ds_port_forward(self, instance_nb=0):
             raise Exception('Port-forward for pod %s on port %s not ready after %ss' %
                             (ds_pod_name, ds_local_port, duration))
 
-    @staticmethod
```

Comment:
Curious, why did you remove this and opt for the utils.cmd()?

Reply:
ah, nvm.  I see you moved it to a lib

Reply:
Yep I had to create a new method for foreground processes